improver/cli/wind_direction.py

```diff
@@ -65,5 +65,5 @@ def process(wind_direction: cli.inputcube,
     from improver.wind_calculations.wind_direction import WindDirection
 
     result, _, _ = (
-        WindDirection(backup_method=backup_method).process(wind_direction))
+        WindDirection(backup_method=backup_method)(wind_direction))
```

Comment:
Is there a better way to phrase this call, or is this "best practise" style?  (Eg `result, = WindDirection(...` or `result, ... = WindDirection(...`?)  This just looks horrible to me.

Reply:
I've stuck a [0] index on the end of the call which means we simply don't assign the [1] and [2] elements rather than assigning them to null. Is this preferred? It is perhaps a little more hidden with that little [0] tucked at the end.

Reply:
I thought the underscore way of doing it was a standard python thing, but I might be wrong.